fighthealthinsurance/models.py

```
@@ -2216,6 +2216,117 @@ def __str__(self):
+        "OngoingChat",
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="policy_analyses",
+    )
+
+    class Meta:
+        indexes = [
+            models.Index(fields=["policy_document"]),
+            models.Index(fields=["created_at"]),
+        ]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)